package.json

```diff
@@ -13,10 +13,11 @@
   "dependencies": {
     "@bufbuild/connect-web": "^0.6.0",
     "@grpc/grpc-js": "^1.7.3",
-    "@openfeature/js-sdk": "~1.0.0",
+    "@openfeature/js-sdk": "^1.1.0",
     "@opentelemetry/api": "^1.3.0",
     "@protobuf-ts/grpc-transport": "2.8.2",
     "axios": "1.2.3",
+    "lru-cache": "^7.14.1",
```

Comment:
@beeme1mr I've verified this is properly added to the deps of the `package.json` in the flagd dist.